lls/main.c

```diff
@@ -500,12 +531,12 @@ lls_stage2(void *arg)
 	 * Gatekeeper or Grantor.
 	 */
 
-	/*
-	 * TODO Have a different block set up RSS on the back interface,
-	 * and classify + distribute ND packets to the LLS block using
-	 * the DPDK libraries.
-	 */
-	if (lls_conf->nd_cache.iface_enabled(net_conf, &net_conf->back)) {
+	ret = register_nd_acl_rules(&net_conf->front);
+	if (ret < 0)
+		return ret;
+
+	/* TODO Have a different block set up RSS on the back interface. */
```

Comment:
I'm not sure I see the motivation for this. Should `finalize_stage2()` make sure the RSS on the front is set as well? It seems like there's no reason to assume the LLS wouldn't set it if the back interface is enabled, and the RT part of the GK block will fulfill this role a couple of patches down the line anyway.